interfaces/repo.go

```diff
@@ -609,6 +624,45 @@ func (r *Repository) securitySnippetsForSnap(snapName string, securitySystem Sec
 	return snippets, nil
 }
 
+// SnapSpecification returns the specification of a given snap in a given security system.
+func (r *Repository) SnapSpecification(securitySystem SecuritySystem, snapName string) (Specification, error) {
+	r.m.Lock()
+	defer r.m.Unlock()
+
+	backend := r.backends[securitySystem]
+	if backend == nil {
+		return nil, fmt.Errorf("cannot obtain specification of snap %q, security system %q is not known", snapName, securitySystem)
```

Comment:
"cannot obtain specification of snap" => "cannot handle interfaces of snap"

We don't have the concept of a _"snap specification"_ by itself.. it's actually the _"interface specification for a snap".._ it's okay for the method to be _"SnapSpecification"_ though, because this is an _"interface repository"._ We should probably also avoid telling the user too much about _"interface specifications"_ since this is too much of an implementation detail.